docs/ConfigurationFile.md

```diff
@@ -47,6 +47,24 @@ validation:
     iou_threshold: 0.4
     val_accuracy_interval: 20
 ```
+## Passing config arguments at runtime using a dict
+
+It can often be useful to pass config args directly to a model instead of editing the config file. By using a dict with that matches with the config keys, main.deepforest will update the config after reading from file.
+
+```
+m = main.deepforest()
+assert not m.config["num_classes"] == 2
```

Comment:
I think the use of `assert` statements here could be be confusing for some users. Any reason not to just show the result with `m.config["num_classes"]` and equivalent throughout?